components/popover/popover-mixin.js

```diff
@@ -418,7 +418,9 @@ export const PopoverMixin = superclass => class extends superclass {
 
 		};
 
+		content.style.overflow = 'hidden';
 		const scrollWidth = content.scrollWidth;
+		content.style.removeProperty('overflow');
```

Comment:
I've found that the two column example I added doesn't work great on any browser when scrollbars are always visible when popover is on. It looks like previously dropdown content was adding an extra 18px but with popover it's only 3 and this could be causing there to not be enough space. Let me know thoughts on this. I didn't want to mess with it too much.

Reply:
Oh wow. Hmmm, it sounds like this is something we need to address...? I mean eventually `DropdownContentMixin` will be deleted, so the behaviour should be identical unless there is an existing bug we've chosen to live with.

Reply:
For sure. Should I open up a story for that?

Reply:
I've created a story [here](https://desire2learn.atlassian.net/browse/GAUD-8054) for this.